modules/resource-logs-viewing-cli-console.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * logging/resource-logs-viewing.adoc
+
+[id="resource-logs-viewing-cli-console_{context}"]
+= Viewing resource logs
+
+You can view the log for various resources in the {product-title} CLI and web console. Logs read from the tail, or end, of the log.
+
+.Prerequisites
+
+* Access to the OpenShift CLI (oc).
+
+.Procedure (UI)
+
+. In the {product-title} console, navigate to *Workloads* -> *Pods* or navigate to the pod through the resource you want to investigate. 
++
+[NOTE]
+====
+Some resources, such as builds, do not have pods to query directly. In such instances, you can locate the Logs link on the Details page for the resource.
+====
+
+. Select a project from the drop-down menu.
+
+. Click the name of the pod you want to investigate.
+
+. Click *Logs*. 
+
+.Procedure (CLI)
+
+** View the log for a specific pod:
++
+[source,terminal]
+----
+$ oc logs -f <pod_name> -c <container_name>
+----
++
+--
+where:
+
+`-f`:: Optional: Specifies that the output follows what is being written into the logs. 
+`<pod_name>`:: Specifies the name of the pod.
+`<container_name>`:: Optional: Specifies the name of a container. When a pod has more than one container, you must specify the container name.
+--
++
+For example:
++
+[source,terminal]
+----
+$ oc logs ruby-58cd97df55-mww7r
+----
++
+[source,terminal]
+----
+$ oc logs -f ruby-57f7f4855b-znl92 -c ruby
+----
+
+** View the log for a specific resource:
++
+[source,terminal]
+----
+$ oc logs <object_type>/<resource_name> <1>
+----
+<1> Specifies the resource type and name.
++
+For example:
++
+[source,terminal]
+----
+$ oc logs deployment/ruby
+----
++
+The contents of log files are printed out.
```

Comment:
"The contents of log files are printed out." -- I'm assuming that this applies to the first bullet, too (View the log for a specific pod). What to you think about adding it to the end of the first bullet as well, or removing the `+` so that it renders in line with the two bullet points?